### PR TITLE
Branch final ug

### PR DESCRIPTION
**Pull request by NeoMinWei**
quick fixes

### CODE REVIEW

File: docs/UserGuide.md [hunk 1]
```diff
@@ -598,7 +610,7 @@ Incorrect format of JSON file may lead to:
 - Corrupted file (user will be prompted to repair the file if he wants to)
 - Deletion of stock entries that are erroneous (Financial Planner has a built-in method to remove
 stock entries that does not match the format specified above)
-- Incorrect information printed by Financial Planner application (eg. changing stock prices directly in JSON file)
+- Incorrect information printed by Financial Planner application (e.g. changing stock prices directly in JSON file)
```
Comment (ryan1604) [inline docs/UserGuide.md:613]: eg.!

**Review by ryan1604 — APPROVED** [review]
LGTM

File: docs/UserGuide.md [hunk 2]
```diff
@@ -589,7 +601,7 @@ the application is not running
 
 Example file content of watchlist.json:
 
-![](/images/investments/watchlistjsonexample.png)
+![](images/investments/watchlistjsonexample.png)
```
Comment (wwweert123) [inline docs/UserGuide.md:604]: Thanks!

File: docs/UserGuide.md [hunk 3]
```diff
@@ -853,7 +865,7 @@ window. Sorry for the inconvenience caused. 🥲
 | **Delete cashflow**              | `delete INDEX [/r]`                                        |
 | **Delete income**                | `delete income INDEX [/r]`                                 |
 | **Delete expense**               | `delete expense INDEX [/r]`                                |
-| **Delete recurrence**            | `delete recurrence INDEX [/r]`                             |
+| **Delete recurrence**            | `delete recurring INDEX [/r]`                              |
```
Comment (wwweert123) [inline docs/UserGuide.md:868]: Nice!

**Review by wwweert123 — APPROVED** [review]

**Pull request merged**